pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/ProcessRuntime.java

```diff
@@ -135,6 +146,16 @@
         args.add(String.valueOf(instanceConfig.getFunctionDetails().getProcessingGuarantees()));
         args.add("--pulsar_serviceurl");
         args.add(pulsarServiceUrl);
+        if(isNotBlank(clientAuthenticationPlugin) && isNotBlank(clientAuthenticationParameters)) {
+            args.add("--client_auth_plugin");
+            args.add(clientAuthenticationPlugin);
+            args.add("--client_auth_params");
+            args.add(clientAuthenticationParameters);
+            args.add("--use_tls");
+            args.add(Boolean.toString(useTls));
+            args.add("--tls_allow_insecure");
+            args.add(Boolean.toString(TlsAllowInsecureConnection));
```

Comment:
I think TLS options should be specified even if authentication is disabled.

Reply:
oops..yes, I thought I had fixed in the last commit.. will fix it.